sass/etna.scss

```diff
@@ -59,6 +59,9 @@
 @use "@nationalarchives/frontend/nationalarchives/tools/typography";
 @use "@nationalarchives/frontend/nationalarchives/variables/grid";
 
+// Redeclare $orange to match OHOS orange
+$orange: #FA8334;
+
```

Comment:
I'm not able to see the Orange buttons. I think its due to https://github.com/nationalarchives/ds-ohos-wagtail/actions/runs/6892375564/job/18749511237?pr=10   `fab format`  - updates scss to fix this
After that I think it should be good to view the Orange

Reply:
Thanks, Allan. My editor was treating `.scss` files incorrectly so Prettier wasn't fixing formatting on save. I've now sorted this so we should see fewer formatting issues in future commits